in_toto/models/metadata.py

```diff
@@ -65,7 +65,7 @@ def __repr__(self):
       )
 
 
-  def dump(self, filename):
+  def dump(self, filename, path = ""):
```

Comment:
A couple of things:

1. By convention, there should be no spaces around the `=` for keyword arguments. You'll also see mention of it in the lint report on Travis: https://travis-ci.org/github/in-toto/in-toto/jobs/691965668#L252
2. I don't think we need to take this approach. While the argument is called `filename`, it can accept a full path. While we can consider changing the name of the variable, we can definitely just pass in a full path here, meaning we may not need to make any changes here.